APIs_Platform/FileHandle/main.cpp

```diff
@@ -0,0 +1,20 @@
+#include "mbed.h"
+
+static DigitalOut led2(LED2);
+
+// UARTSerial derives from FileHandle
+static UARTSerial device(STDIO_UART_TX, STDIO_UART_RX);

```

Comment:
I think the UARTSerial will be deprecated soon, can you replace it with the new one:
https://github.com/ARMmbed/mbed-os/blob/master/docs/design-documents/drivers/serial/serial.md

Reply:
Thanks, changed to `BufferedSerial`.